src/client/components/serach-summary/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './index.scss';
+import {Header} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+
+export class SearchSummaryComponent extends React.Component {
+    static defaultProps = {
+        numberOfMatched: 0
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+
+            // destination: this.props.destination,
```

Comment:
Commented out code shouldn't be committed. You can do it to test some changes in your app, but it's only for you alone. Other developers will not understand why you commented it out.